lib/Graph/Graph.cpp

```diff
@@ -1541,6 +1541,32 @@ Node *Function::createWeightedSum(llvm::StringRef name,
   return currAdd;
 }
 
+Node *Function::createClip(llvm::StringRef name, NodeValue input, float min,
+                           float max) {
+  Node *minClipped, *result;
+  if (min != std::numeric_limits<float>::lowest()) {
+    auto *minSplat =
+        createSplat(name.str() + ".minSplat", input.getType(), min);
+    // minClipped = (input <= min) ? min : input
+    auto *LTEMinCond = createCmpLTE(name.str() + ".lteMin", input, minSplat);
+    minClipped =
+        createSelect(name.str() + ".clipMin", LTEMinCond, minSplat, input);
```

Comment:
Is it possible to use `Min` rather than `CmpLTE+Select`?

Reply:
Yes, it is possible. Code updated.